syft/execution/computation.py

```diff
@@ -36,6 +36,15 @@ def __init__(self, name, target, args_, kwargs_, return_ids):
         self.kwargs = kwargs_
         self.return_ids = return_ids
 
+    def copy(self):
```

Comment:
I know I'm not supposed to touch `Action`s but having a copy method makes it a bit easier to translate list of actions. I hope this is a reasonable compromise :)

Reply:
Yeah, this is a good addition.